helm/odigos/templates/odigos-deployment.yaml

```
@@ -2,8 +2,9 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: odigos-deployment
-  namespace: {{ .Release.Namespace }}
+  namespace: { { .Release.Namespace } }
 data:
-  ODIGOS_VERSION: {{ .Values.image.tag | default .Chart.AppVersion }}
-  ODIGOS_TIER: "{{- if .Values.onPremToken }}onprem{{- else }}community{{- end }}"
+  ODIGOS_VERSION: { { .Values.image.tag | default .Chart.AppVersion } }
```

When a space is introduced between curlys (due to prettier) - the variable will not be pulled.
Please wrap the curlys with quotes:
```suggestion
  ODIGOS_VERSION: '{{ .Values.image.tag | default .Chart.AppVersion }}'
```